src/content/overview/fleet-management/app-management/index.md

```diff
@@ -0,0 +1,135 @@
+---
+linkTitle: App management
+title: App management
+description: The app platform allows to manage app catalogs and apps, for simple and standardized deployment across the platform.
+weight: 40
+menu:
+  principal:
+    parent: overview-fleet-management
+    identifier: overview-fleet-management-app-platform
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+user_questions:
+  - What does App Platform do?
+  - What is an App Catalog?
+  - What is a Managed App?
+  - Does Giant Swarm provide app catalogs out of the box?
+  - How is App Platform implemented?
+  - How does the App Platform relate to Helm?
+  - What are the components of App Platform?
+  - What are the app catalogs that Giant Swarm provides out of the box?
+  - How can I create an organizational app catalog?
+  - How can I interact with the Giant Swarm App Platform?
+  - Can I create an app catalog?
+last_review_date: 2024-10-22
+---
+
+The _Giant Swarm App Platform_ refers to a set of features and concepts that allow you to browse, install and manage the configurations of apps (such as `Prometheus`) from a single place; the [platform API]({{< relref "/overview/architecture#platform-api" >}}).
```

Comment:
There is a mixture of casing of app names when enclosed as inline code that feels off. Here you use `Prometheus`, in the next paragraph `Helm`, yet lower down in the doc `coredns` and `cluster-autoscaler` are referred as lowercase variants.

Generally I am of the opinion that app names should be lowercase when referred to as inline code, reflecting the casing you would use as part of a command, however opinions may vary but we should be consistent in the casing we choose.

Reply:
We agreed on using lowercase; sadly, I have trouble finding and replacing it. Looking at this